src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,159 @@
+    Distractor1: "It's a new JavaScript framework",
+    Distractor2: "It's a sorting algorithm",
+    Distractor3: "It's a type of programming paradigm",
+    Explanation: 
+    "The extension aids in inspecting and ensuring web accessibility by providing relevant information in the Developer tools.",
+    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/"
+  },
+  {
+    Question: "Which tool or extension allows you to find accessibility defects on your web page in both Chrome and Firefox?",
+    Answer: 
+    "WAVE",
+    Distractor1: "Color Oracle",
+    Distractor2: "aXe",
+    Distractor3: "ChromeVox",
+    Explanation: 
+    "WAVE is an extension available for both Chrome and Firefox that helps identify accessibility issues on web pages.",
+    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/"
+  },
+  {
+    Question: "What is the primary function of a screen reader in web development?",
+    Answer: 
+    "Screen readers are used to convey the content of a webpage to users with visual impairments via text-to-speech, helping developers understand and create accessible web content.",
+    Distractor1: "It's a web design tool",
+    Distractor2: "It's a code editor",
+    Distractor3: "It's a graphics rendering software",
+    Explanation: 
+    "Screen readers assist users with visual impairments in accessing web content through spoken text.",
+    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/"
+  },
+  {
+    Question: "Why is it important to add labels to form inputs like search forms?",
+    Answer: 
+      "Labels indicate the purpose of form inputs and make them accessible to screen reader users.",
+    Distractor1: "Labels improve website aesthetics.",
+    Distractor2: "Labels increase website loading speed.",
+    Distractor3: "Labels provide translations for form inputs.",
+    Explanation: 
+      "Labels associated with form inputs, such as search forms, describe the purpose of the input, making it accessible to screen reader users.",
+    Link: "https://www.freecodecamp.org/news/common-accessibility-errors-and-how-to-fix-them/"
+  },  
+  {
+    Question: "What are the four broad guidelines of web accessibility according to WCAG?",
+    Answer: 
+      "Perceivable, Operable, Understandable, Robust",
+    Distractor1: "Creative, Original, Unique, Dynamic",
+    Distractor2: "Left, Right, Up, Down",
+    Distractor3: "HTML, CSS, JavaScript, PHP",
+    Explanation: 
+      "The four broad guidelines of web accessibility are Perceivable, Operable, Understandable, and Robust, often referred to as WCAG.",
+    Link: "https://www.freecodecamp.org/news/what-is-website-accessibility-18ce00ec990f/"
+
+  },
+  {
+    Question: "Which screen reader is a Free & Open-source downloadable software for Windows OS?",
```

Comment:
add the year, since things can change in the future 

```suggestion
    Question: "As of 2023, which screen reader is a Free & Open-source downloadable software for Windows OS?",
```